Exercise4.sh

```diff
@@ -0,0 +1,36 @@
+#Q1
+#write a file containing unique gender and years experience in two space deliminated columns
+#input = wages.csv
+#output uni.gen.exp.csv
+cat wages.csv | cut -d ',' -f1,2 | sed 's/\<[0-9]\>/0&/' | sed 's/,/ /g' > a.tmp.csv
+head -1 a.tmp.csv > u.gender.year.csv
+sed '1d' a.tmp.csv |  sort -n | uniq >>u.gender.year.csv
+rm a.tmp.csv
+echo "unique gender and experience .csv complete"
+
```

Comment:
Alternatively, you can use:

cat wages.csv | grep -v gender | cut -d , -f 1,2 | sort -u | tr ',' ' ' | sort -k1,1d -k2,2n >> categories.txt